### ISSUE TITLE
af-magic improvment

### ISSUE THREAD

**Issue by leoheck** [robbyrussell/oh-my-zsh#5249]
Where can I ask a feature for `af-magic` theme?

Considering it from here, [af-magic.zsh-theme](https://github.com/robbyrussell/oh-my-zsh/blob/master/themes/af-magic.zsh-theme), I would like to know if its is possible to replace the **line 9** by this:

``` bash
SPACER=$(printf '-%.0s' {1..$(tput cols)})
PROMPT='$FG[241]$SPACER%{$reset_color%}
```

This will create a resizable separator line instead of being using a fixed one.
Thanks!

**Comment by okapia** [comment]
A simpler way to create the line is:

```
SPACER="${(l.COLUMNS..-.)}"
```

**Comment by mcornella** [comment]
Implemented in #7739. Go +1 it if it works for you.
